### PR TITLE
chore: update getting-started.md

### PR DESCRIPTION
**Pull request by parid0kht**
## Description
fixing command in `website/docs/getting-started.md`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore (change that has absolutely no effect on users)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: website/docs/getting-started.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ npx lerna run test --scope=header
 ```
 
 ```
-> lerna run test --scope=header
+> npx lerna run test --scope=header
```
Comment (erictuvesson) [inline website/docs/getting-started.md:231]: The output in this case will be, it's just lerna that is echoing the command you were running.

```
> test
> lerna run test --scope=header
```

**Review by JamesHenry — CHANGES_REQUESTED** [review]
Thanks @parid0kht, but @erictuvesson's feedback is exactly correct, please revert that change

File: website/docs/getting-started.md (hunk 1 above)
Comment (parid0kht) [reply website/docs/getting-started.md]: I revert my change, thanks

**Comment by JamesHenry** [comment]
Thanks @parid0kht!

**Pull request merged**